src/app/test/test.component.html

```diff
@@ -108,21 +115,20 @@
                   }
                 </td>
                 <td>
-                  @if (getReranReport(report.storageId)) {
+                  @if (report.reranReport) {
                     <span data-cy-test="runResult"
-                          [id]="'runResult#' + report.storageId"
-                          [style.color]="getReranReport(report.storageId)!.color">
-                      {{ getReranReport(report.storageId)!.resultString }}
+                          [style.color]="report.reranReport.color">
+                      {{ report.reranReport.resultString }}
                     </span>
                   }
                 </td>
                 <td>
-                  @if (getReranReport(report.storageId)) {
+                  @if (report.reranReport) {
                     <span class="buttons">
                         <button data-cy-test="replaceReport" class="btn btn-info" title="Replace report"
-                                (click)="replaceReport(report.storageId)">Replace</button>
+                                disabled (click)="replaceReport(report)">Replace</button>
```

Comment:
Is this disabled because the functionality to replace a report is not implemented yet?

Reply:
Yes, I added disabled to the button and added a toast warning to the method that is called by the button to make it extra clear

Reply:
Do we have an issue for that already? Maybe in the backend?

Reply:
Yes #544